chain_info/Moonbase.yaml

```
@@ -0,0 +1,54 @@
+name: Moonbase Alpha
+network_type: testnet
+para_id: 1000
+chain:
+  type: parachain
+  parent: Moonbase Relay (based on Westend)
+  consensus: POS
+  sudo: true
```

The format has changed and para_id slides down inside `chain`.

```suggestion
chain:
  type: parachain
  parent: Moonbase Relay (based on Westend)
  consensus: POS
  sudo: true
  para_id: 1000
```